src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  key: string | undefined
+}
+
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
```

Comment:
TS will infer this type from initial object
```suggestion
  state = {
```

Reply:
also rename the variable to `keyPressed` as per task description